GridList/GridList.jsx

```diff
@@ -0,0 +1,178 @@
+import { h } from "preact";
+import MaterialComponent from "../MaterialComponent";
+
+/**
+ * @prop cellHeight	{number} - Number of px for one cell height. You can set 'auto' if you want to let the children determine the height.
+ * @prop cols {number} - Number of columns.
+ * @prop padding {1|4} - Number of px for the padding/spacing between items.
+ * @prop style {object} - Override the inline styles of the root element
+ */
+class GridList extends MaterialComponent {
+  constructor() {
+    super();
+    this.componentName = "grid-list";
+    this._propsDict = {
+      cols: "cols",
+      padding: "padding",
+      cellHeight: "cellHeight",
```

Comment:
where is cellHeight used?

Reply:
Not yet.  There are a lot of unused props at this point because I assumed we would want to mirror the interface from Material UI (http://www.material-ui.com/#/components/grid-list) and so I stubbed out the props they supported.